src/test/resources/features/wishlistFunctionality.feature

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
delete blank space and rename feature file as specified in PR #12

Reply:
Done